src/main/packages/uml-communication-diagram/uml-communication-link/uml-communication-link-update.tsx

```diff
@@ -154,15 +154,15 @@ class CommunicationLinkUpdate extends Component<Props, State> {
 
   private rename = (value: ICommunicationLinkMessage) => (name: string) => {
     const { element, update } = this.props;
-    const messages: ICommunicationLinkMessage[] = [...element.messages];
+    const messages: ICommunicationLinkMessage[] = element.messages.map((message) => ({ ...message }));
```

Comment:
this is a change in behaviour. the previous syntax would have used the same elements, while this code also clones the items. @matthiaslehnertum is that intended?

Reply:
With the instance being inserted into the store directly before, we had the issue that the editor itself would alter properties on the reference to the "serialized" element which triggers assertions within Redux as this is non-intended behavior. I therefore introduced the copy-logic to prevent such issues. 🤔